src/styles/block/cards.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  width: 100%;
+  display: grid;
+  padding-top: 50px;
+  margin: 0 auto;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  place-content: center;
+
+  &__section {
+    width: 200px;
+    padding: 32px 16px 16px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: $background-color;
+    transition: transform 0.3s ease;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__header {
+    font-family: Roboto, sans-serif;
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-color;
+  }
+
+  &__info {
+    margin-top: 24px;
+    display: flex;
+    font-display: row;
+    justify-content: space-between;
+  }
+
+  &__content {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $second-color;
+  }
+
+  &__raiting {
+    margin-top: 16px;
+    display: flex;
+    font-display: row;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    color: $main-color;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__sale {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__link {
+    margin-top: 16px;
+    height: 40px;
+    width: 166px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    border-radius: 5px;
+    background-color: $blue-color;
+    font-family: Roboto, sans-serif;
+    line-height: 16px;
+    font-weight: 700;
+    font-size: 14px;
+    color: $cardTextButtonColor;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-position: center;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--5 span:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
The `nth-child` selector here is set to `-n + 4` which means it will select no children, because `-n` is always less than 4. You probably meant to write `nth-child(-n+5)` to select up to the 5th child.